src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -21,11 +26,22 @@ namespace Bloom.web
 	/// thread-safe.</remarks>
 	public class EnhancedImageServer: ImageServer
 	{
+		// A string unlikely to occur in a book name which we can substitute for a single quote
+		// when making a book path name to insert into JavaScript.
+		private const string QuoteSubstitute = "$xquotex$";
```

Comment:
I can try...AFAIK it just has to not contain an apostrophe and be something very unlikely to occur in a real book name.